presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -819,3 +819,26 @@ row. The returned value is ``NULL`` if the subquery produces no rows::
     WHERE regionkey = (SELECT max(regionkey) FROM region)
 
 .. note:: Currently only single column can be returned from the scalar subquery.
+
+LATERAL
+^^^^^^^
+Subqueries appearing in the FROM clause can be preceded by the key word LATERAL.
+This allows them to reference columns provided by preceding FROM items.
+
+A LATERAL item can appear at top level in the FROM list, or within a JOIN tree.
```

Comment:
How about
```
A ``LATERAL`` join can appear at the top level in the ``FROM`` list, or anywhere
within a parenthesized join tree. In the latter case, it can also refer to any items
that are on the left-hand side of a ``JOIN`` that it is on the right-hand side of.